Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,50 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
```

Comment:
Edit this a bit more. 

> The code will be responsible for controlling the hardware and manipulating data.

Not specific.

> All the data and instructions must flow through the microcontroller giving a common place for all of the data to go and be readily available for whichever subsystem has need of it.

Does that need to be said? Does it explain what the subsystem does?

State what this is to do very specifically.

Reply:
Fluff removed.